src/main/webapp/utilities/CONSTANTS.js

```diff
@@ -0,0 +1 @@
+const API_KEY = ''; // your API key
```

Comment:
Please add todo to generate this file during build/setup.

Reply:
Added it!